### PR TITLE
jnethack 3.6.7-0.1

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
`brew audit` need some change (as the repo has name `-alpha` in it 😅 )

**Comment by chenrui333** [comment]
release repo should resolve the audit failure issue

File: Formula/j/jnethack.rb [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
   # We use a git checkout to avoid patching the upstream NetHack tarball.
-  url "https://scm.osdn.net/gitroot/jnethack/source.git",
-      tag:      "v3.6.6-0.6",
-      revision: "1fe57469d3f3bc20997f5f219c33bf4973a41b82"
+  url "https://github.com/jnethack/jnethack-release.git",
+      tag:      "v3.6.7-0.1",
+      revision: "3b3a9c4e25df60f9bce2ad09ce368410b4360e85"
   license "NGPL"
-  head "https://github.com/jnethack/jnethack-alpha.git", branch: "develop"
+  head "https://github.com/jnethack/jnethack-release.git", branch: "master"
```
Comment (ZhongRuoyu) [inline Formula/j/jnethack.rb:9]: Why this change? It seems development does indeed happen in the `jnethack-alpha` repo (see `jnethack-release`'s repo description).

Comment (chenrui333) [reply Formula/j/jnethack.rb:9]: the change was made after this comment, https://github.com/jnethack/jnethack-alpha/issues/11#issuecomment-1795894988

Comment (chenrui333) [reply Formula/j/jnethack.rb:9]: I saw the two repos are quite similar, the release might the one used specially for release.

**Review by iMichka — APPROVED** [review]
